### PR TITLE
Googlesheets plugin update (version 1.1.0)

### PR DESCRIPTION
**Pull request by jereze**

### CODE REVIEW

File: googlesheets/custom-recipes/googlesheets-append/recipe.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        },
+        {
+            "name": "insert_format",
+            "label": "Values interpretation",
+            "description": "How data is interpretated by the spreadsheet",
+            "type": "SELECT",
+            "selectChoices": [
+                {
+                    "value": "RAW",
+                    "label": "RAW: not parsed, stored as-is (default)"
```
Comment (alexbourret) [inline googlesheets/custom-recipes/googlesheets-append/recipe.json:58]: Is there a point keeping this option ? Is there any difference between this and turning all columns into strings before syncing ?

Comment (jereze) [reply googlesheets/custom-recipes/googlesheets-append/recipe.json:58]: Google may a use a different format in the sheet depending on this value. And we want to keep the previous behavior as the default one for stability.

File: googlesheets/custom-recipes/googlesheets-append/recipe.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import dataiku
+from dataiku.customrecipe import *
```
[the remark this answers is not in the document]
Comment (jereze) [reply googlesheets/custom-recipes/googlesheets-append/recipe.py]: Yes. Resolved.

File: googlesheets/python-lib/googlesheets.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    except gspread.exceptions.WorksheetNotFound as e:
+        raise Exception("Trying to open non-existent sheet. Verify that the sheet name exists (%s)." % tab_id)
+    except gspread.exceptions.APIError as e:
+        if hasattr(e, 'response'):
+            error_json = e.response.json()
+            print(error_json)
+            error_status = error_json.get("error", {}).get("status")
+            email = credentials.get("client_email", "(email missing)")
+            if error_status == 'PERMISSION_DENIED':
+                raise Exception("The Service Account does not have permission to read or write on the spreadsheet document. Have you shared the spreadsheet with %s?" % email)
```
[the remark this answers is not in the document]
Comment (jereze) [reply googlesheets/python-lib/googlesheets.py]: Thanks! Resolved.

**Pull request merged**